components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -667,8 +671,8 @@ void AdsServiceImpl::Initialize() {
                           base::Unretained(this)));
 
   profile_pref_change_registrar_.Add(
-      kBraveTodayOptedIn, base::BindRepeating(&AdsServiceImpl::OnPrefsChanged,
-                                              base::Unretained(this)));
+      kNewTabPageShowToday, base::BindRepeating(&AdsServiceImpl::OnPrefsChanged,
```

Comment:
shouldn't this monitor _both_ `kBraveTodayOptedIn` and `kNewTabPageShowToday`?

Reply:
Fixed